development-docs/DEV_GUIDE.md

```diff
@@ -78,6 +78,9 @@ structure after everything is said and done should look like `/Library/Java/Java
 doing that run the command at the beginning again and this should link the file and allow you to use maven with OpenJDK
 version 11.
 
+When running the tests, you may encounter `openssl` related errors for parts that you may not have even worked on, in 
+which case please consider upgrading your `openssl` version.
+
```

Comment:
It might be good to try to figure this out since nobody else reported any issues. What version of OpenSSL are you using?

Reply:
I was using 2.8.3 when I had the issues.
Now that I'm using the OpenSSL 3.0.5 it works fine.

Reply:
I don't think that openSSL has any verion 2.8. Are you on Mac? In that case, it might have been LibreSSL which MacOS defaults to. So maybe we can be more specific here and say that you should make sure you are using OpenSSL and not LibreSSL?